src/main/java/me/botsko/prism/Il8n.java

```diff
@@ -0,0 +1,80 @@
+package me.botsko.prism;
+
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.TextComponent;
+import net.kyori.adventure.util.UTF8ResourceBundleControl;
+import org.jetbrains.annotations.PropertyKey;
+
+import java.util.MissingFormatArgumentException;
+import java.util.MissingResourceException;
+import java.util.ResourceBundle;
+
+/**
+ * Created for the Prism-Bukkit Project.
+ * Created by Narimm on 29/07/2020.
+ */
+public class Il8n {
```

Comment:
Codacy found an issue: [The utility class name 'Il8n' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=6279951)